ext/standard/info.h

```diff
@@ -52,13 +52,12 @@
 
 BEGIN_EXTERN_C()
-PHPAPI zend_string *php_info_html_esc(char *string);
-PHPAPI void php_info_html_esc_write(char *string, int str_len);
```

Comment:
I can not find any implementation of `php_info_html_esc_write `

Reply:
The definition of `php_info_html_esc_write()` has been removed in PHP 5.4, so the declaration can be removed as well.